fava/default-settings.conf

```diff
@@ -28,6 +28,8 @@ journal-show-type-note = True
 journal-show-type-document = True
 journal-show-type-pad = False
 journal-show-type-query = False
+journal-show-type-custom = True
+journal-show-type-budget = True
```

Comment:
OK I see the point of not wasting the space for a toggle button that might not be used at all by the user.

So what do you think about only showing it when budget directives are used?